apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts

```diff
@@ -124,4 +124,7 @@ export const contentConfig: ConfigurableContent = {
     },
     resourceIndexLabel: 'Available Resources',
     defaultLanguageCode: LanguageCode.Chilcotin,
+    phoneNumber: '123456789',
+    address: '',
+    internalLinks: [],
```

Comment:
It's very important to provide examples for such things in the `.SAMPLE` config.

Reply:
Populate a few internal links with varying logo dimensions. We can make the links all point to `www.coscrad.org` and load a few different sample logos in the media gallery there.